docs/gatsby-node.js

```diff
@@ -17,14 +17,22 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
 exports.createPages = async ({ graphql, actions, reporter }) => {
   const { createPage } = actions;
   const result = await graphql(`
-    query AllGuides {
+    query GuidesQuery {
       allMdx(filter: { fields: { collection: { eq: "guides" } } }) {
         nodes {
+          id
+          body
+          slug
           frontmatter {
             title
+            date
+            image {
+              id
+              absolutePath
+              name
+              size
+            }
```

Comment:
I'm not sure that these fields are necessary in page context either. They can be queried in the pages themselves.